webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = {
+  entry: path.resolve(__dirname, 'client', 'index.jsx'),
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'public'),
+  },
+  resolve: {
+    extensions: ['.js', '.jsx', '.json', '.css', '.scss'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
```

Comment:
I think you may be missing a character in this regex. Take a second look at the [webpack: babel-loader docs](https://webpack.js.org/loaders/babel-loader/)

```suggestion
        test: /\.m?(js|jsx)$/,
```

Reply:
The one in my config is the one I got from Jenny, and for the last couple of projects, everything has transpiled properly.  I also noticed that it is inconsistent with babel's page, but it has worked thus far, so I keep using it.  A quick search, and I noticed it shows up in other people's web packs.